### PR TITLE
Fix/ci pull master release

### PR DESCRIPTION
**Pull request by brunocampos-ssa**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the packaging and release process by adding a new job step to create a zip file for the `kos-js` directory.
	- Updated the release creation command to utilize the newly packaged zip file.

- **Improvements**
	- Upgraded the `webpack` dependency to the latest major version, introducing potential optimizations and new features for the build process.
	- Improved security by modifying the handling of the `.env` file in the GitHub Actions workflow.

### CODE REVIEW

File: .github/workflows/pull-develop.yaml [hunk 1]
```diff
@@ -47,8 +47,9 @@ jobs:
           cargo pants
 
       - name: Restore .env file
-        run: |
-          echo "${{ vars.NODES_ENV }}" > packages/kos-sdk/.env.nodes
+        run: echo $NODES_ENV | base64 -d > packages/kos-sdk/.env.nodes
+        env:
+          NODES_ENV: ${{ secrets.NODES_ENV }}
```
[the remark this answers is not in the document]
Comment (brunocampos-ssa) [reply .github/workflows/pull-develop.yaml]: thanks Mr. Rabbit, I fixed the issue.

**Review by pedrxlz — APPROVED** [review]

File: .github/workflows/pull-master-release.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
           with_cache: true
 
       - name: Restore .env file
-        run: echo $NODES_ENV | base64 -d > packages/kos-sdk/.env.nodes
+        run: echo "$NODES_ENV" | base64 -d > packages/kos-sdk/.env.nodes
         env:
           NODES_ENV: ${{ secrets.NODES_ENV }}
```
[the remark this answers is not in the document]
Comment (brunocampos-ssa) [reply .github/workflows/pull-master-release.yaml:36]: Thanks for the recommendation, but this `.env.nodes` does not contain any sensitive information. It is URLs from public blockchain nodes services.

@coderabbitai resolve

File: .github/workflows/pull-master-release.yaml [hunk 3]
```diff
@@ -82,7 +85,7 @@
       - name: Create GitHub Release
         run: |
           VERSION=$(cat VERSION)
-          gh release create "v$VERSION" ./demo/kos --title "Release v$VERSION" --notes "${{ steps.release_notes.outputs.notes }}"
+          gh release create "v$VERSION" ./demo/kos-js.zip --title "Release v$VERSION" --notes "${{ steps.release_notes.outputs.notes }}"
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
[the remark this answers is not in the document]
Comment (brunocampos-ssa) [reply .github/workflows/pull-master-release.yaml]: Fixed.

File: .github/workflows/pull-master-release.yaml [hunk 4]
```diff
@@ -72,6 +72,9 @@
       - name: Build
         run: make webpack-npm
 
+      - name: Package kos-js
+        run: zip -r ./demo/kos-js.zip ./demo/kos
+
```
[the remark this answers is not in the document]
Comment (brunocampos-ssa) [reply .github/workflows/pull-master-release.yaml]: fixed.

**Review by pedrxlz — APPROVED** [review]

**Pull request merged**